brand/models/commentary.py

```diff
@@ -1,13 +1,35 @@
+import json
+
 from django.core.exceptions import ValidationError
 from django.core.validators import MaxValueValidator, MinValueValidator
 from django.db import models
 
 import yaml
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError as YamlValidationError
+from yamlfield.fields import YAMLField
 
 from brand.models import Brand
 from brand.models.embrace_campaign import EmbraceCampaign
 
 
+def validate_feature_override_yaml(feature_yaml) -> None:
+    # Ensure feature_yaml format is of dict/yaml type
+    if not isinstance(feature_yaml, dict):
+        raise ValidationError(f"Only yaml format is accepted")
+
+    with open("./config/harvest_validation_schema.json", "r") as harvest_json_file:
```

Comment:
This is locking the application with i/o. It probably won't kill it but still isn't good practice. Every time the thing runs it has to re-open and close a file. Considering that the JSON isn't that big, it's better to just open the .json at the top of the file and then keep it in memory.

The issue isn't enough to request changes for, but just a heads up